translate/convert/po2json.py

```diff
@@ -40,7 +40,7 @@ def convertstore(self, includefuzzy=False, remove_untranslated=False):
         self.remove_untranslated = remove_untranslated
         self.inputstore.makeindex()
         for unit in self.templatestore.units:
-            inputunit = self.inputstore.locationindex.get(unit.getid())
+            inputunit = self.inputstore.locationindex.get(str(unit.getid()))
```

Comment:
Can't `unit.getid()` be altered to return a string?

Reply:
That is certainly an option, this was mostly posted to get such kind of feedback - where to put the abstraction.

Reply:
Please note I've left a more elaborated comment on https://github.com/translate/translate/issues/3819#issuecomment-449043811